meetings/membership-criteria.md

```diff
@@ -0,0 +1,64 @@
+# Membership criteria
+
+When you apply for membership, we review your application based on the [membership participation types](membership-participation-types.md) defined. We ask for some basic information about you. 
+
+## Basic information
+
+First, we validate the personal information:
+
+- First and Last name are reviewed to eliminate potential 'bots'.
+- A valid email address: We only validate that it is a valid email address format. Later, when an invitation is sent, that verifies that it's a real address.
+- A valid GitHub ID: We validate the profile appears to be human. There should be some activity. The user name must match any contribution links provided. GitHub activity mentioned in the contribution statement should match this ID.
+
+## Open Source Maintainer, Open Source Contributor
```

Comment:
Question for the project committee. Could projects be hosted on another open platform? e.g. GitLab, BitBucket.

Reply:
Shaun should comment on this one. I don't think it's come up yet, however.